app/workers/form1010cg/submission_job.rb

```diff
@@ -1,21 +1,40 @@
 # frozen_string_literal: true
 
+require 'sidekiq/monitored_worker'
+
 module Form1010cg
   class SubmissionJob
+    STATSD_KEY_PREFIX = "#{Form1010cg::Auditor::STATSD_KEY_PREFIX}.async."
     include Sidekiq::Worker
+    include Sidekiq::MonitoredWorker
     include SentryLogging
 
     sidekiq_options(retry: 14)
 
+    sidekiq_retries_exhausted do |msg, _e|
+      StatsD.increment("#{STATSD_KEY_PREFIX}failed_no_retries_left", tags: ["claim_id:#{msg['args'][0]}"])
+    end
+
+    def retry_limits_for_notification
+      [10]
+    end
+
+    def notify(params)
+      StatsD.increment("#{STATSD_KEY_PREFIX}failed_ten_retries", tags: ["params:#{params}"])
```

Comment:
I believe there is a missing .

 StatsD.increment("#{STATSD_KEY_PREFIX}.failed_ten_retries",